src/Compilers/CSharp/Test/Semantic/Semantics/NullCheckedParameterTests.cs

```diff
@@ -152,7 +156,10 @@ class C
 {
     public string this[int index!] => null;
 }";
-            CreateCompilation(source).VerifyDiagnostics();
+            var comp = CreateCompilation(source);
+            comp.VerifyDiagnostics();
+            var m = comp.GlobalNamespace.GetTypeMember("C").GetMember<SourcePropertySymbol>("this[]");
+            Debug.Assert(((SourceParameterSymbol)m.Parameters[0]).IsNullChecked == true);
```

Comment:
is the left side of this `==` comparison a nullable? If not consider removing the `== true` and using `!` instead of `== false`

Reply:
Looks like I clicked on a stale notification and this was already fixed.